tools/tsp-client/README.md

```diff
@@ -2,45 +2,65 @@
 
 A simple command line tool for generating TypeSpec clients.
 
-### Usage
+### Installation
 ```
-tsp-client [options] <outputDir>
+npm install @azure-tools/typespec-client-generator-cli
```

Comment:
Since this doc is in `azure-sdk-tools`, it's fine to show how to install `latest`.  But if users are running this tool in a managed repo like `azure-rest-api-specs` or `azure-sdk-for-lang`:

1. The tool version should be pinned in a `package.json` and `package-lock.json`
2. Users should install the tool by running `npm ci` (assuming there is a lockfile)
3. Users should run the tool using `npx <tool-name>` (which uses the version installed from lockfile)

Reply:
I think this needs more thought on how we want language repositories to expose this to users. Added a tracking issue here: https://github.com/Azure/azure-sdk-tools/issues/7169